libs/wcm/utils/requestHandlers.js

```diff
@@ -0,0 +1,70 @@
+/* istanbul ignore file */
+// @todo Add test coverage by #4418
+import { formatJsonRpcError, formatJsonRpcResult } from '@json-rpc-tools/utils';
+import { getSdkError } from '@walletconnect/utils';
+import { transactions, cryptography } from '@liskhq/lisk-client';
+import { ERROR_CASES } from '../constants/lifeCycle';
+import { SIGNING_METHODS } from '../constants/permissions';
+
+const signMessage = async (message, wallet) => {
```

Comment:
For now to complete your PR i understand we need these two functions, but we will have just one implementation of signMessage and signTransaction in the entire codebase, so can you add a TODO here to remove these functions and use the generic functions?

Reply:
Sure. I'll fix this by the next PR